v2/internal/controllers/crd_networking_privatelinkservice_test.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	. "github.com/onsi/gomega"
+
+	network20201101 "github.com/Azure/azure-service-operator/v2/api/network/v1beta20201101"
+	network20220701 "github.com/Azure/azure-service-operator/v2/api/network/v1beta20220701"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1beta20200601"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_Networking_PrivateLinkService_CRUD(t *testing.T) {
+	t.Parallel()
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	vnet := newVMVirtualNetwork(tc, testcommon.AsOwner(rg))
+
+	privateLinkServiceNetworkPolicies := network20201101.SubnetPropertiesFormat_PrivateLinkServiceNetworkPolicies_Disabled
+	subnet := newVMSubnet(tc, testcommon.AsOwner(vnet))
+	subnet.Spec.PrivateLinkServiceNetworkPolicies = &privateLinkServiceNetworkPolicies
+
+	lb, frontendIPConfigurationARMID := newLbForPLS(tc, rg, subnet)
```

Comment:
```suggestion
	lb, frontendIPConfigurationARMID := newLoadBalancerForPLS(tc, rg, subnet)
```

I think one acronym per method is probably ideal =)